server/venueless/live/modules/room.py

```diff
@@ -169,3 +169,38 @@ async def push_room_info(self, body):
                 ),
             ]
         )
+
+    @command("schedule")
+    @room_action(permission_required=Permission.ROOM_ANNOUNCE)
+    async def change_schedule_data(self, body):
+        data = body.get("schedule_data")
+        if data and not all(key in ["title", "session"] for key in data.keys()):
+            raise ConsumerException(
+                code="room.unknown_schedule_data", message="Unknown schedule data"
+            )
+
+        await self.consumer.send_success({})
+        self.room.schedule_data = data
+        await update_room(self.room)
+        await self.consumer.channel_layer.group_send(
+            GROUP_ROOM.format(id=self.room.pk),
+            {"type": "room.schedule", "schedule_data": data, "room": str(self.room.pk)},
+        )
+
+    @event("schedule", refresh_world=True)
+    @room_action()
```

Comment:
From reading the code and logs, this room action is the only thing I can imagine being the problem with the failing tests, it shouldn't be required. Otherwise I have to dig deeper.